plugins/lightspeed/package.json

```diff
@@ -40,7 +40,9 @@
     "@material-ui/icons": "^4.11.3",
     "@material-ui/lab": "^4.0.0-alpha.61",
     "@mui/icons-material": "^5.15.18",
+    "@patternfly/react-core": "^5.3.4",
```

Comment:
I think we should use the latest alpha for PF V6. This will help in integration with PF Chatbot Extensions since that is built on V6.